devtools/gradle/src/main/java/io/quarkus/gradle/QuarkusPlugin.java

```diff
@@ -249,7 +249,7 @@ private void setupQuarkusBuildTaskDeps(Project project, Project dep, Set<String>
 
         final Task quarkusDev = project.getTasks().findByName(QUARKUS_DEV_TASK_NAME);
         if (quarkusDev != null) {
-            final Task resourcesTask = dep.getTasks().findByName(JavaPlugin.PROCESS_RESOURCES_TASK_NAME);
+            final Task resourcesTask = dep.getTasks().findByName("jandex");
```

Comment:
"jandex" should not be replacing "processResources" here. There should be an additional `if` block for "jandex".

Reply:
Okk